scripts/biospecimen_count.Rmd

```diff
@@ -161,33 +167,68 @@ if ( any(is.na(rna_counts$stage)) ) {
 
 <br>
 
-#### Combine Count Tables
+### Combine Count Tables
 
 Combine the count tables, organize them for display, and save.
 
----
+```{r}
+rbind(dna_counts, rna_counts) %>%
+# change cohort names to be in line with the language on the Pediatric 
+# Molecular Targets website
+  mutate(cohort = case_when(cohort == 'DGD' ~ 'CHOP P30 Panel',
+                            cohort == 'TARGET' & 
+                              experimental_strategy == 'Targeted Sequencing' ~ 
+                              'TARGET Panel',
+                            TRUE ~ cohort)) -> dataset_disease_counts
+```
+
+#### Per Dataset and Disease Counts
+
+```{r}
+dataset_disease_counts %>%
+# TCGA and CHOP P30 DNA Panel data are not currently being used, so filter out
+  filter(cohort != 'TCGA', !is.na(cancer_group),
+         !(cohort == 'CHOP P30 Panel' & sequencing_type == 'DNA')) %>%
+# change column names to be in line with the language on the Pediatric 
+# Molecular Targets website and retain desired columns
+  select(Dataset = cohort, `Sequencing Type` = sequencing_type,
+        `Tumor Stage` = stage, Disease = cancer_group, 
+        `Number of Biospecimens` = n) %>%
+  arrange(Dataset, Disease, `Sequencing Type`, 
+          `Tumor Stage`) -> dataset_disease_counts_filtered
+
+# save as a plain text table
+write_csv(dataset_disease_counts_filtered, 
+          'biospecimen_dataset_disease_counts.csv')
```

Comment:
I don't like this syntax, I prefer to have a separate object and write function so I don't accidentally overwrite the table when I'm playing with or updating the code

Reply:
:) the `#` before the `%>%` works wonders! But that is also fine, personal preference for saving tables separately. It is easier to follow with the variable at the top though, instead of the end and our codebase in the analysis repo follows this style.

Reply:
The printing table at the end is the only thing you'd need to do to have it show and be scrollable in the html notebook

Reply:
I figured as long as I was updating, I'd use `DT::datatable()` so it would be filterable as well. This is on the list for updates in https://github.com/PediatricOpenTargets/ticket-tracker/issues/462